SS_popdyn.tpl

```diff
@@ -324,7 +328,7 @@ FUNCTION void get_initial_conditions()
 
   #ifdef DO_ONCE
   if (do_once == 1)
-    cout << " ready for virgin age struc " << endl;
+    echoinput << "Begin calculating virgin age struc " << endl;
```

Comment:
I would think that here and below on line 434 that if you have a statement that starts with "Begin" you would want a subsequent line in the code that has "Finished"?

Reply:
I agree that would be a better style, but these statements are only in the code to assist in debugging when a weird result is occurring.  The delineation is implied when the next section begins.